src/main/java/com/firebolt/jdbc/statement/FireboltStatement.java

```diff
@@ -134,6 +133,8 @@ private void abortStatementRunningOnFirebolt(String statementId) throws SQLExcep
 				statementService.abortStatement(statementId, this.sessionProperties);
 			}
 			log.debug("Statement with id {} was aborted", statementId);
+		} catch (FireboltException e) {
+			throw e;
```

Comment:
is this not redundant?

Reply:
No because just below we do 
catch (Exception e) {
....

We want to throw a FireboltException if an exception that is not a FireboltException is found. If the exception is already a FireboltException then we simply re-throw it

Reply:
Ok I see